### ISSUE TITLE
cant load checkpoint files

### ISSUE THREAD

**Issue by skywo1f** [GeorgeSeif/Semantic-Segmentation-Suite#225]
tried all of the files in the checkpoints folder:
model.ckpt.index
model.ckpt.meta
checkpoint
model.ckpt.data-00000-of-00001
none of them work:
Semantic-Segmentation-Suite/checkpoints/0295/model.ckpt.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator

**Comment by sweetboxwwy** [comment]
Have you solve this problem?

**Comment by K-M-Ibrahim-Khalilullah** [comment]
@skywo1f @GeorgeSeif 
same problem. Anyone suggest me please.
 I trained on my PC , training is Ok but can not load the checkpoint file

**Comment by millermuttu** [comment]
**same issue -** 


NotFoundError (see above for traceback): Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Key BatchNorm/beta not found in checkpoint
         [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
         [[Node: save/RestoreV2/_79 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_84_save/RestoreV2", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

**Comment by FZY2019** [comment]
Is the problem solved？

**Comment by skywo1f** [comment]
no

**Comment by Imperssonator** [comment]
Had the same issue - in your case, you should just pass `model.ckpt`
https://stackoverflow.com/questions/33759623/tensorflow-how-to-save-restore-a-model
Also if you're not using CamVid, make sure to pass something to `--dataset`, otherwise it will default to the 32 class labels from the CamVid dataset. Hope that helps.

**Comment by wy9884255** [comment]
you better try this disk:/your_folder/model.ckpt

**Comment by Harikrishnan24** [comment]
Is this problem solved

**Comment by mtylerpreston** [comment]
I think I have the solution...just use `model.ckpt` even if no such file exists.

I had the same struggle in trying to pass model.ckpt.meta etc in for resuming the train. Even though no file name exists in the directory where the checkpoint was specified to be saved, I just used `model.ckpt` and it worked out.

**Comment by Harikrishnan24** [comment]
Thank you for your help
